api/server/server.go

```diff
@@ -912,27 +937,59 @@ func (s *Server) bindHandlers(ctx context.Context) {
 				apps.PUT("/routes/*route", s.handleRoutesPostPut)
 			}
 
+			cleanv2 := engine.Group("/v2")
+			v2 := cleanv2.Group("")
+			v2.Use(s.apiMiddlewareWrapper())
+
+			{
+				v2.GET("/apps", s.handleAppList)
+				v2.POST("/apps", s.handleAppCreate)
+				v2.GET("/apps/:appId", s.handleAppGet)
```

Comment:
in the process of linting... would appreciate doing `appID` / `fnID` / etc. while you're in here (lint will yell), though I can do a sweep afterwards for a low low price for you my friend *merchant voice*

Reply:
yep